### DISCUSSION TITLE
Rate my Repository in Professionality

### DISCUSSION THREAD

**Discussion by ghost** [community/community#88547] [Programming Help] [answered]
### Body

Hi guys I recently created a Repository called `369-CSS-EFFECTS`... So i want to create a perfect/better repository that is it should be more appealing or professional in nature.. Suggest me some ways to do it and rate the existing repository...and if you could you can also contribute. Pls help me with the above mentioned things...

[Link](https://github.com/Yuheshpandian/369-CSS-EFFECTS) to the Repository...

Thank You all in advance

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by LaFocus** [comment, marked as answer]
I guess your repo looks pretty good! Im loved. I liked the fact that there is good looking documentation with the list of all effects with their discription & usage paragraph in there. Good Job!

**Reply by ghost** [reply]
Thank You so much for your kind suggestions. Your words motivated me to create more projects like. If you have any more ideas to improve the project structurally you can share or even contribute if you are willing.

**Reply by ghost** [reply]
_Also thank for your appreciations, I would sure follow the same...☺️_

**Comment by ghost** [comment]
Also can someone suggest me some github actions/apps that might be useful for this repository. For now I am using greeting workflow...
